src/coreclr/jit/compiler.cpp

```diff
@@ -3413,6 +3413,12 @@ bool Compiler::compJitHaltMethod()
     /* This method returns true when we use an INS_BREAKPOINT to allow us to step into the generated native code */
     /* Note that this these two "Jit" environment variables also work for ngen images */
 
+    // if (JitConfig.JitHalt().contains(info.compMethodHnd, info.compClassHnd, &info.compMethodInfo->args) /*&&
+    // opts.jitFlags->IsSet(JitFlags::JIT_FLAG_TIER1)*/)
+    //{
+    //    return true;
+    //}
+
```

Comment:
Remove. Or add `JitHaltTier1`?

Reply:
it was not needed and I forgot to remove the commented code.